aspnetcore/fundamentals/localization.md

```diff
@@ -194,7 +194,7 @@ Localization is configured in the `ConfigureServices` method:
 
 ### Localization middleware
 
-The current culture on a request is set in the localization [Middleware](middleware.md). The localization middleware is enabled in the `Configure` method of *Program.cs* file. Note, the localization middleware must be configured before any middleware which might check the request culture (for example, `app.UseMvcWithDefaultRoute()`).
+The current culture on a request is set in the localization [Middleware](xref:fundamentals/middleware/index). The localization middleware is enabled in the `Configure` method of *Program.cs* file. Note, the localization middleware must be configured before any middleware which might check the request culture (for example, `app.UseMvcWithDefaultRoute()`).
```

Comment:
~~in the `Configure` method of *Program.cs* file~~
in *Startup.Configure*

Reply:
It's ok in this case :point_right: https://github.com/aspnet/Docs/blob/master/aspnetcore/fundamentals/localization/sample/Localization/Program.cs#L130

The samp will prob go back to a `Startup` class on the 2.1 update.

Reply:
... but it is missing the definite article there. Hold on a sec ... I'll fix that.